packages/starlight/utils/translations.ts

```diff
@@ -19,10 +19,20 @@ try {
 const defaults = buildDictionary(
 	builtinTranslations.en!,
 	userTranslations.en,
-	builtinTranslations[defaultLocale],
+	builtinTranslations[stripLangRegion(defaultLocale)],
```

Comment:
Oh actually, one thought: what if we had a built-in language with a regional specifier. We would no longer match it here right?

Thinking for example of Traditional Chinese. If that was built in at `zh-TW` and a user uses `lang: zh-TW`, it would get truncated and they’d get our `zh` strings instead I think?

Reply:
Maybe this kind of approach?
```suggestion
	builtinTranslations[defaultLocale] || builtinTranslations[stripLangRegion(defaultLocale)],
```

Reply:
Good catch, yeah, I think that should do the trick!

Reply:
Great. And then I guess the same applies the second place we use `stripLangRegion` too.